npm test fails

Just following straight instructions I get this. This is an admittedly old node, 8.12

```
npm test

> typescript-action@0.0.0 test /home/jmerelo/Code/js/typescript-action
> jest

 FAIL  __tests__/main.test.ts
  ● Test suite failed to run

    Jest encountered an unexpected token

    This usually means that you are trying to import a file which Jest cannot parse, e.g. it's not plain JavaScript.

    By default, if Jest sees a Babel config, it will use that to transform your files, ignoring "node_modules".

    Here's what you can do:
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/en/configuration.html

    Details:

    /home/jmerelo/Code/js/typescript-action/node_modules/jest-circus/node_modules/jest-matcher-utils/build/index.js:136
      } catch {
              ^

    SyntaxError: Unexpected token {

      at ScriptTransformer._transformAndBuildScript (node_modules/@jest/transform/build/ScriptTransformer.js:537:17)
      at ScriptTransformer.transform (node_modules/@jest/transform/build/ScriptTransformer.js:579:25)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        0.039s
Ran all test suites.
npm ERR! Test failed.  See above for more details.
```

Latest version. 14, does not have a problem (it issues another series of warnings though)

--- a/README.md
+++ b/README.md
@@ -16,6 +16,8 @@ Click the `Use this Template` and provide the new repo details for your action
 
 ## Code in Main
 
+> First, you'll need to have a reasonably modern version of `node` handy. This won't work with versions older than 9, for instance.
+
 Install the dependencies  
 ```bash
 $ npm install

